### ISSUE TITLE
Looks like jscritic.com is owned by someone else?

### ISSUE THREAD

**Issue by LMLB** [kangax/jscritic#35]

**Comment by XhmikosR** [comment]
@kangax it seems the domain was bought by someone else...

**Comment by kangax** [comment]
Hmm, yeah I probably forgot to renew it. I don't think there's much value in this ancient project anymore? I wonder if removing CNAME would make https://kangax.github.io/jscritic work at least

**Comment by XhmikosR** [comment]
Yup, it should and agreed. Just keep it around for historical reasons :)

**Comment by rotu** [comment]
@kangax It’s still listed as a “current” project and linked from https://kangax.github.io/. You may want to update that page.
